<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>811</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>English Meaning</string>
     </attribute>
     <widget class="QTextEdit" name="englishTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>271</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tamil Meaning</string>
     </attribute>
     <widget class="QTextEdit" name="tamilTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>271</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Tamil" country="India"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="synonymList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>811</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Synonyms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterWordEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>Enter word</string>
    </property>
    <property name="accessibleDescription">
     <string>Enter the word you want to search for</string>
    </property>
    <property name="placeholderText">
     <string>Enter Word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>Search Word</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
